### ISSUE TITLE
WordPress.com Email Regarding Scheduled Posts Limit

### ISSUE THREAD

**Issue by synora** [Automattic/wp-calypso#73933]
### Quick summary

We’ve had some users report receiving the following email:

```
Hi there,

You are receiving this message because you have scheduled more than 400 posts on your WordPress.com site at whichmeamitoday.wordpress.com. We currently need to limit the number of scheduled posts per site. Any posts you scheduled beyond the first 400 will not publish automatically.

We apologize for any inconvenience this causes you. Placing posts in draft status and scheduling/publishing them later is a workable, alternate method.

If you have any questions about this limit, feel free to contact us at help@wordpress.com.

Thank you.
```

These users only have 13-16 posts scheduled. I’m not finding any common pattern with these users. All are on different plans:  2 Business (one Atomic and one not Atomic), Premium, and Personal.

### Steps to reproduce

It's unclear how this email was triggered especially because these users only have 13-16 posts scheduled at the time of this report. 

I've asked this one for more info: [6003557](6003557-zd-woothemes)

### What you expected to happen

For an email about a scheduled post limit to not happen as these users are well under the scheduled post limit of 100.

### What actually happened

Somehow, this email was triggered for some users. 

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple, Atomic

**Comment by synora** [comment]
User reports: 

[6003557](6003557-zd-woothemes)
[6003917](6003917-zd-woothemes)
[6003201](6003201-zd-woothemes)
[6002639](6002639-zd-woothemes)

**Comment by wiesenhauss** [comment]
41122240-hc

**Comment by synora** [comment]
6004248-zen

**Comment by synora** [comment]
6004602-zen
6005285-zen

**Comment by dsas** [comment]
Deployed a fix in D103175-code
